src/sssom_schema/schema/sssom_schema.yaml

```
@@ -76,12 +76,30 @@ enums:
         meaning: sssom:NegatedPredicate
   mapping_cardinality_enum:
     permissible_values:
-      "1:1": One-to-one mapping
-      "1:n": One-to-many mapping
-      "n:1": Many-to-one mapping
-      "1:0": One-to-none mapping
-      "0:1": None-to-one mapping
-      "n:n": Many-to-many mapping
+      "1:1":
+        description: Indicates the mapping record is about a one-to-one mapping, that is,
+          the subject and the object are only mapped to each other, exclusive of any
+          other subject or object.
+      "1:n":
+        description: Indicates the mapping record is about a one-to-many mapping, that
+          is, the same subject is mapped to several different objects.
+      "n:1":
+        description: Indicates the mapping record is about a many-to-one mapping, that
+          is, several different subjects are mapped to the same object.
+      "n:n":
+        description: Indicates the mapping record is about a many-to-many mapping, that
+          is, the subject is mapped to several different objects and the object is
+          mapped to several different subjects.
+      "1:0":
+        description: Indicates that the subject has no match in the object vocabulary.
+          This value MUST only be used when the object_id is sssom:NoTermFound.
+      "0:1":
+        description: Indicates that the object has no match in the subject vocabulary.
+          This  value MUST only be used when the subject_id is sssom:NoTermFound.
```

```suggestion
          This value MUST only be used when the subject_id is sssom:NoTermFound.
```
Minor typo - delete extra space.